CHANGELOG.md

```diff
@@ -7,9 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Added
 ### Changed
+- remove pandas due to too old debian version [i112](https://github.com/greenbone/pheme/pull/112)
 ### Deprecated
 ### Removed
 - libsass support: https://sass-lang.com/blog/libsass-is-deprecated [111](https://github.com/greenbone/pheme/pull/111)
+- equipment treemap [i112](https://github.com/greenbone/pheme/pull/112)
```

Comment:
i112?

Reply:
for go into insert mode and then call an ambulance, removed i :)